Configuration/AlCa/python/autoAlca.py

```diff
@@ -2,7 +2,7 @@
                   'Charmonium'     : 'TkAlJpsiMuMu',
                   'Commissioning'  : 'HcalCalIsoTrk',
                   'Cosmics'        : 'TkAlCosmics0T+MuAlGlobalCosmics+HcalCalHOCosmics+DtCalibCosmics',
-                  'DoubleEG'       : 'EcalCalZElectron+EcalUncalZElectron+HcalCalIterativePhiSym+HcalCalIsoTrkFilter',
+                  'DoubleEG'       : 'EcalCalZElectron+EcalUncalZElectron+EcalTrg+HcalCalIterativePhiSym+HcalCalIsoTrkFilter',
```

Comment:
@shervin86 I think the correct approach would be to use an AlCaRecoTriggerBit to silence it at will.